src/format/parser.d

```diff
@@ -2503,6 +2511,14 @@ private:
 		assert(0);
 	}
 
+	void parseShortenedFunctionBody() in(match(TokenType.FatArrow)) {
+		nextToken();
+		space();
+		parseExpression();
+		nextToken();
+		newline(2);
+	}
```

Comment:
I think you might have a splice a span somewhere in there so that thing preferably break on the =>